sdk/spring/README.md

```diff
@@ -147,6 +148,45 @@ If you’re a Maven user, add our BOM to your pom.xml `<dependencyManagement>` s
 ```
 [//]: # ({x-version-update-end})
 
+## Spring Boot 3 Support
+
+The source code of Spring Cloud Azure for Spring Boot 3.x can be found on the [feature/spring-boot-3](https://github.com/Azure/azure-sdk-for-java/tree/feature/spring-boot-3) branch.
+
+#### Disable the JAR signature verification for the native image generation
+
+You have to disable the JAR signature verification to be able to generate a native image ([see](https://github.com/Azure/azure-sdk-for-java/issues/30320)).
```

Comment:
Azure SDK JARs are signed. [Spring Boot 3 does not support today signed JARs](https://github.com/Azure/azure-sdk-for-java/issues/30320) when you run your application with [AOT mode on a JVM](https://docs.spring.io/spring-boot/docs/current/reference/html/deployment.html#deployment.efficient.aot) or you [build an native image](https://docs.spring.io/spring-boot/docs/current/reference/html/native-image.html).